functions/Get-DbaLastGoodCheckDb.ps1

```diff
@@ -0,0 +1,183 @@
+﻿Function Get-DbaLastGoodCheckDb
```

Comment:
What makes it good or bad? The code suggests you're just looking for the datetime value of the last checkdb? Wouldn't that just be Get-DbaLastCheckDb?